guides/v2.3/graphql/reference/categories.md

```diff
@@ -40,17 +40,19 @@ Attribute | Data type | Description
 `product_count`| Int | The number of products in the category
 `default_sort_by`| String | The attribute to use for sorting
 `products(<attributes>)` | `CategoryProducts` | The list of products assigned to the category
-`children` | `CategoryTree` |
+`breadcrumbs` | `Breadcrumb` |
```

Comment:
Breadcrumb description missing because it's obvious, right?

Further down page:  Breadcrumb object ---    Make Breadcrumb link to the Breadcrumb object

or in description:  A Breadcrumb object that contains a set of links to show customers where they are in relation to other ....